Headers/QuartzCore/CALayer.h

```diff
@@ -30,34 +30,34 @@
 #import "QuartzCore/CABase.h"
 #import "QuartzCore/CAMediaTiming.h"
 #import "QuartzCore/CATransform3D.h"
-#if GNUSTEP
+//#if GNUSTEP
```

Comment:
Either leave the #if in place or remove it completely. In the later case you will have to add some explanation why.
The same is true for all similar changes you made.